backend/ops_api/ops/resources/research_type.py

```diff
@@ -1,6 +1,12 @@
+from flask import Response
 from models.research_projects import ResearchType
 from ops_api.ops.base_views import EnumListAPI
+from ops_api.ops.utils.auth import is_authorized
+from typing_extensions import override
 
 
 class ResearchTypeListAPI(EnumListAPI, enum=ResearchType):  # type: ignore [call-arg]
-    pass
+    @override
+    @is_authorized("GET_RESEARCH_PORJECT", "GET_RESEARCH_PROJECTS")
```

Comment:
would be better if these were enums: `GET_RESEARCH_PORJECT`

Reply:
I thought about that as well. Went back and forth a lot on it. Might be worth making them enums for clarity and some safety from mis-typing.

Reply:
I decided for now to make it 1 permission per endpoint. As we didn't need anything more than that. If we ever do, we can revisit this question and come with a good solution.